### ISSUE TITLE
Automatic Installer Not Working

### ISSUE THREAD

**Issue by jcarney2024** [OS4ED/openSIS-Classic#265]
The openSIS 8.0 automatic installer is **not working**, stuck at blank page after Database Connection step. After you hit connect, the page goes blank, and you can't do anything.

**Comment by openSISAdmin** [comment]
What's the OS you are using?

**Comment by jcarney2024** [comment]
Windows Server 2021

**Comment by openSISAdmin** [comment]
Go to: https://www.apachefriends.org/download.html

Download and install this:

![image](https://user-images.githubusercontent.com/7434280/165631935-f2b4c5e1-c785-4b85-8967-fc5b088679e3.png)


Watch this video. It works for both PC and server version on Windows>

https://www.youtube.com/watch?v=a8aB0GSGMlc

**Comment by Leozom** [comment]
> The openSIS 8.0 automatic installer is **not working**, stuck at blank page after Database Connection step. After you hit connect, the page goes blank, and you can't do anything.

same experience

**Comment by FunkyMonk8111** [comment]
what is the root pswd? I'm getting stuck at the db connection screen.

**Comment by qpwr** [comment]
@FunkyMonk8111 how did you do that? I don't even have fields to put the database data:
<img width="728" alt="Bildschirmfoto 2022-09-09 um 14 49 36" src="https://user-images.githubusercontent.com/3735390/189369441-7e5901f4-8d49-469c-b012-c972f053abf9.png">

**Comment by FunkyMonk8111** [comment]
@qpwr  Go to your host or hosting software and create just a sample database, it can be anything - just an empty database. Then, take your username and password and put it in the box above. All you need in this step is just a database to connect to.

**Comment by jdanielnd** [comment]
I'm having the same issue here. I've created an empty database and it still goes to the blank screen.

Actually, if I enter a wrong credential it raises an error. The blank screen only appears if the database connection information is correct, what makes me think that OpenSIS is connecting to the database but stops there.

I've tried with V7.6 and the behavior is the same. Any ideas?

**Comment by qpwr** [comment]
For me the reason was that *many* of the files actually had whitespace after the closing tag, which caused errors because headers could not be sent ("Headers already sent by..."). I'd suggest to enable error_reporting to check where the errors are coming from. I've got the installer up and running by fixing the whitespace issue, but ran into more issues after finishing the installation.

Unfortunately I discovered while debugging this that the overall code quality is subpar, to say it politely. A *lot* of code duplication and a lot of spaghetti code, just to name two examples. Not blaming anyone, but I'd say that this project is not the way to go for new installations.

I decided to head from this project due to these issues.
